cheatsheets/Authentication_Cheat_Sheet.md

```diff
@@ -270,6 +270,21 @@ UAF takes advantage of existing security technologies present on devices for aut
 
 U2F augments password-based authentication using a hardware token (typically USB) that stores cryptographic authentication keys and uses them for signing. The user can use the same token as a second factor for multiple applications. U2F works with web applications. It provides **protection against phishing** by using the URL of the website to lookup the stored authentication key.
 
+## Autonomous TOTP
+
+The Time-based One-Time Password algorithm (TOTP) is an extension of the HMAC-based One-time Password algorithm (HOTP) generating a one-time password by instead taking uniqueness from the current time. It has been adopted as Internet Engineering Task Force[1] standard RFC 6238, is the cornerstone of Initiative For Open Authentication (OATH), and is used in a number of [two-factor authentication](https://en.wikipedia.org/wiki/Two-factor_authentication) systems.
+
+A server is needed to generating token for TOTP Value, the client must have same logic with their authentication server, so *authorized client* should be always send a valid TOTP Value
+
+Autonomous TOTP must meet several requirements, namely:
+
+1. token are created by machine dynamically (autonomously), otherwise you need to store primary token safely
+2. connection with low latency is needed
```

Comment:
Is this true? AFAIK TOTP tokens are usually valid for 30s, and you can normally use one token on either side, so there's a reasonable amount of leeway. Perhaps rephrase as something like `The time of the two systems needs to be relatively closely in sync.`?